jest.config.js

```diff
@@ -0,0 +1,4 @@
+// jest.config.js
+module.exports = {
```

Comment:
Any reason you prefer a separate file? Not sure which way I like better

Reply:
I just felt that this doesn't belong to `package.json`, most of our tools configs are outside in their own files, so made sense to me to put it in their own file.